### ISSUE TITLE
make build - command fails for Ansible

### ISSUE THREAD

**Issue by dacodekid** [kartoza/docker-geosafe#112]
Trying to follow the [Ansible based installation](https://github.com/kartoza/docker-geosafe/blob/develop/deployment/ansible/README.md) for `GeoNode with QGIS Server Backend and GeoSAFE`, the `make build` command fails with following error.

```sh
╰─# make build

------------------------------------------------------------------
Building in production mode
------------------------------------------------------------------
data_dir_conf uses an image, skipping
rabbitmq uses an image, skipping
postgis uses an image, skipping
geoserver uses an image, skipping
Building celery
Step 1/22 : FROM geonode/geonode:geonode
# Executing 4 build triggers
ERROR: Service 'celery' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder226993340/requirements.txt: no such file or directory
Makefile:29: recipe for target 'build' failed
make: *** [build] Error 1
```

**Comment by dacodekid** [comment]
Found the issue.

----------
`requirements.txt: no such file or directory`
-----------
The repository named all `requirements[-XXXXXX].txt` as `REQUIREMENTS[-XXXXXX].txt`, (which I think might work on `MacOS` as it won't enforce case in file names).

Changed all file names & update all `Dockerfile` to lowercase `requirements` and it started building `celery`.  However it ended with another build error.

```sh
Directory '/usr/src/app/' is not installable. File 'setup.py' not found.
ERROR: Service 'celery' failed to build: The command '/bin/sh -c pip install --no-deps --no-cache-dir -e /usr/src/app/' returned a non-zero code: 1
Makefile:29: recipe for target 'build' failed
make: *** [build] Error 1
```

**Comment by lucernae** [comment]
@dacodekid I'm afraid that is different file.

the file `requirements.txt` you mentioned is owned by geonode repo (it is on src/geonode). The step before `make build` is `make build-geonode-core`. Is it working correctly or returning an error? GeoNode repo uses build trigger on its base image, so we need to build the image first. The error you found at first means the previous geonode image were not built correctly, so it ends up trying to build the image in the wrong location (and not found the requirements.txt file, because it is in the wrong context). This requirements.txt file is located in src/geonode and will be included automatically when you execute `make build-geonode-core`.
That means we should check the output of `make build-geonode-core` first to find the issue.

**Comment by Gustry** [comment]
> the file requirements.txt you mentioned is owned by geonode repo (it is on src/geonode). The step before make build is make build-geonode-core.

FYI, a big PR has been merged yesterday in geonode master: https://github.com/GeoNode/geonode/pull/3610

I don't know if it affects us

**Comment by dacodekid** [comment]
@lucernae / @Gustry 

`make build-geonode-core`  worked fine without any issue. 

I really didn't dig deep into see where the error came from (too late night) but noticed while the error message said `requirements.txt` not found, but all files named `REQUIREMENTS`.  Renaming those file names worked though.  I'll further test this repo in a few -  perhaps it's development build via `pycharm`.

Just out of curiosity, does `travis` build passes for you with `GeoNode`'s recent merge?

**Comment by dacodekid** [comment]
I noticed in logs that during my last provision  (`Debian 9.3`) there were few packages got `404` (and lots of lost packets) and happened on couple other tries as well. Changed to `Ubuntu 16.04` and all went well.

Facing another issue on the `geosafe/analysis/create` page (screenshot below). But since that is unrelated to this ticket, closing it out.  Thx again for your help/time.

![image](https://user-images.githubusercontent.com/10047844/37184035-262fca30-2307-11e8-8f65-1d27b8ed63b4.png)
